### DISCUSSION TITLE
How to make it so that when I make changes to the main branch, Vercel does not trigger?

### DISCUSSION THREAD

**Discussion by okineadev** [vercel/community#6836] [Help] [answered]
### Summary

**Every time** when I make changes in `main` branch, **Vercel** reacts to it and sculpts its error cross, and I don't want that

I want Vercel to build **only** when I make changes to the `website` branch

![image](https://github.com/vercel/community/assets/81070564/a7e22dd5-f68a-41b2-9ffd-2fe843309524)

### Steps to Reproduce

.

**Comment by versecafe** [comment, marked as answer]
@okineadev If you go to `vercel.com/[team]/[project]/settings/git` you can change the production branch

<img width="1312" alt="image" src="https://github.com/vercel/community/assets/147033096/c1b98b89-d9e9-4f7b-be55-fee347ecb5e4">

and then you can set to only deploy for production

<img width="1312" alt="image" src="https://github.com/vercel/community/assets/147033096/38f861dd-c03d-4bae-a901-358ef4537475">

**Comment by okineadev** [comment]
@versecafe, I did all that but it doesn't help :(

![image](https://github.com/vercel/community/assets/81070564/5ca56d7e-2764-4e94-a9bb-b4657d5ffe8c)

**Reply by versecafe** [reply]
@okineadev can you share some screenshots showing that your production branch is set correctly and ignore build steps is correctly configured as sometimes Vercel can have some issues with saving the settings if you're on an unstable connection, just refresh the page and double check, if that doesn't work I can schedule a call when you're free to help you through it if you'd like.

**Comment by okineadev** [comment]
> @okineadev can you share some screenshots showing that your production branch is set correctly and ignore build steps is correctly configured as sometimes Vercel can have some issues with saving the settings if you're on an unstable connection, just refresh the page and double check, if that doesn't work I can schedule a call when you're free to help you through it if you'd like.

Here:

![image](https://github.com/vercel/community/assets/81070564/6678aef3-a268-42af-aa4a-400d7c439bed)
![image](https://github.com/vercel/community/assets/81070564/ea45098e-eff6-4c92-881d-041471b0c091)
